pyaptamer/datasets/_loaders/_aptacom_loader.py

```diff
@@ -0,0 +1,77 @@
+__author__ = "rpgv"
+__all__ = ["load_aptacom"]
+
+import pandas as pd
+from datasets import load_dataset
+
+
+def load_aptacom(as_df=False, filter_entries=None):
+    """
+    Loads complete aptacom dataset from hugging face datasets.
+
+    Parameters
+    ----------
+    as_df: (bool) Requires pandas compatible format; converts
```

Comment:
defaults should be stated in the docstring, and please adopt a consistent formatting. Suggestion:

`paramname : type/description, default=default_value`